EU4toV2/Data_Files/blankMod/output/events/postColonial.txt

```diff
@@ -1,296 +1,293 @@
-country_event = {  id = 251052  title = "EVTNAME251052"  desc = "EVTDESC251001"  major = yes  picture = "Riot"    trigger = {   is_independant = yes   capital_scope = {    OR = {     continent = europe     continent = asia    }   }   NOT = { has_global_flag = RGS_independence }   has_global_flag = der_postcolonialism   is_possible_vassal = RGS   NOT = { exists = RGS }  }    mean_time_to_happen = {   months = 48   modifier = {    factor = 0.75    year = 1845   }   modifier = {    factor = 0.75    year = 1855   }   modifier = {    factor = 0.75    year = 1860   }   modifier = {    factor = 0.5    year = 1865   }  }    immediate = {   set_global_flag = RGS_independence   any_owned = { limit = { is_core = RGS } secede_province = RGS remove_core = THIS }   RGS = {    leadership = 80    years_of_research = 5    set_country_flag = buro_independence    election = yes   }  }   option = {   name = "EVTOPTA251001"   create_vassal = RGS   ai_chance = { factor = 25 }  }    option = {   name = "EVTOPTB251001"   war = {    target = RGS    attacker_goal = {     casus_belli = conquest    }       defender_goal = {     casus_belli = become_independent    }   }   ai_chance = { factor = 75 }  } }
-country_event = {  id = 251053  title = "EVTNAME251053"  desc = "EVTDESC251001"  major = yes  picture = "Riot"    trigger = {   is_independant = yes   capital_scope = {    OR = {     continent = europe     continent = asia    }   }   NOT = { has_global_flag = SON_independence }   has_global_flag = der_postcolonialism   is_possible_vassal = SON   NOT = { exists = SON }  }    mean_time_to_happen = {   months = 48   modifier = {    factor = 0.75    year = 1845   }   modifier = {    factor = 0.75    year = 1855   }   modifier = {    factor = 0.75    year = 1860   }   modifier = {    factor = 0.5    year = 1865   }  }    immediate = {   set_global_flag = SON_independence   any_owned = { limit = { is_core = SON } secede_province = SON remove_core = THIS }   SON = {    leadership = 80    years_of_research = 5    set_country_flag = buro_independence    election = yes   }  }   option = {   name = "EVTOPTA251001"   create_vassal = SON   ai_chance = { factor = 25 }  }    option = {   name = "EVTOPTB251001"   war = {    target = SON    attacker_goal = {     casus_belli = conquest    }       defender_goal = {     casus_belli = become_independent    } 
```

Comment:
It looks like this either incorporates multiple changes, or somehow dropped some other ones (Oranje, Transvaal, and Natalia are missing here, plus many other changes later). Can you double-check this file? If those changes are intended, can you put them in a separate PR for the sake of tracking?

Reply:
Fixed, I had used an outdated base